Avatar/avatar.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Avatar</title>
+    <link rel="stylesheet" href="../utility.css" />
+    <link rel="stylesheet" href="avatar.css" />
+  </head>
+  <body>
+    <div class="example-div">
+      <!-- PROFILE AVATAR -->
+      <img src="../images/avatar2.jpg" alt="avatar" class="avatar" />
```

Comment:
**Fix:** The images are located in a folder named "IMAGES", but here you've used "images" to refer to them. I think this should not work. Like at least it is not working in codesandbox. Just check this and rectify it everywhere.

Reply:
Hi vishal I have rectified the changes can you check once now